contracts/test/avs/eigen/deploy/EigenLayerLocal.sol

```diff
@@ -207,7 +217,22 @@ contract EigenLayerLocal is IEigenDeployer {
                 )
             )
         );
-
         return address(wethStrat);
     }
+
+    function _whitelistStrategies(address[] memory strats) internal {
+        vm.startPrank(strategyManager.strategyWhitelister());
+
+        IStrategy[] memory _strategy = new IStrategy[](strats.length);
+        bool[] memory _thirdPartyTransfersForbiddenValues = new bool[](strats.length);
+
+        for (uint256 i = 0; i < strats.length; i++) {
+            _strategy[i] = IStrategy(strats[i]);
+            _thirdPartyTransfersForbiddenValues[i] = false;
+        }
+
+        strategyManager.addStrategiesToDepositWhitelist(_strategy, _thirdPartyTransfersForbiddenValues);
+
+        vm.stopPrank();
+    }
```

Comment:
Whitelisting all strategies in advance, so saved anvil state has our local strategies whitelisted.

Before, we were only doing this on deposit (if strat not whitelisted, whitelist it, then deposit)